frontend/packages/dev-console/src/components/helm/HelmInstallUpgradePage.tsx

```diff
@@ -87,9 +88,12 @@ const HelmInstallUpgradePage: React.FunctionComponent<HelmInstallUpgradePageProp
 
     const fetchHelmRelease = async () => {
       let res;
+      let error: Error = null;
       try {
         res = await coFetchJSON(config.helmReleaseApi);
-      } catch {} // eslint-disable-line no-empty
+      } catch (e) {
+        error = e;
+      } // eslint-disable-line no-empty
       if (ignore) return;
```

Comment:
just a query. Why do we have this `ignore` check after the coFetch and not before it?

Reply:
Okay I will try to explain it briefly. So we do this to prevent updating a state of an unmounted component and as you can see the state is being updated only after the API call.